contracts/clients/package.json

```diff
@@ -33,6 +33,7 @@
     "mocha": "^10.0.0",
     "source-map-support": "^0.5.21",
     "ts-node": "^10.9.1",
+    "typemoq": "^2.1.0",
```

Comment:
If anyone knows of any other TypeScript friendly mocking libs that would be preferable, let me know.

Reply:
Remove typemoq